ivy_tests/test_ivy/test_frontends/test_torch/test_tensor.py

```diff
@@ -5,6 +5,8 @@
 # local
 import ivy_tests.test_ivy.helpers as helpers
 from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+from ivy_tests.test_ivy.test_frontends.test_torch.test_creation_ops \
+    import _dtypes, _requires_grad
```

Comment:
Seeing as these functions are quite small (basically 5 lines each), I think it would be easier if we just add to this file rather than importing them. 

I believe this would make the code more easy to understand if someone tries to do that! 😅